boto/utils.py

```diff
@@ -862,20 +862,6 @@ def notify(subject, body=None, html_body=None, to_string=None,
             boto.log.exception('notify failed')
 
 
-def get_utf8_value(value):
```

Comment:
This was a public visible method (i.e. not named with a leading underscore) in the API for the `utils` module, meaning users of the Boto library might have been depending on it.

Given this, I'd say we should keep this method here, but add a docstring saying that it's deprecated and has been replaced with `get_utf8able_str` within the Boto library.

Reply:
Makes sense, will do!